islandora_solr_config/includes/admin.inc

```diff
@@ -110,6 +111,12 @@ function islandora_solr_config_admin_rss_settings($form, &$form_state) {
     '#description' => t('Email address for person responsible for technical issues relating to channel.'),
     '#default_value' => $rss_channel['webMaster'] ? $rss_channel['webMaster'] : '',
   );
+  $form['rss_limit'] = array(
+    '#type' => 'textfield',
+    '#title' => 'RSS results limit',
+    '#description' => 'Maximum number of results via RSS.',
+    '#default_value' => $rss_limit,
```

Comment:
You can spare a line and make the '#default_value' => variable_get('islandora_solr_config_rss_limit', 50),

Reply:
Yeah, I was aiming to follow the pattern established through the rest of the admin form. You think it's better to just do the more efficient thing?

Reply:
nah.. if the rest is so, then OK. I'm a minimalist, no worries

Reply:
👍